core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/person/child/AddOrUpdatedChildprofileuseCase.kt

```diff
@@ -0,0 +1,29 @@
+package com.ustadmobile.core.viewmodel.person.child
+
+import com.ustadmobile.lib.db.entities.Person
+
+class AddOrUpdatedChildProfileUseCase {
```

Comment:
I think this is definitely not needed. We had a special use case for course blocks because we need to set the order field and indentation. We are not saving the child profiles in any special order.